example/pubspec.lock

```diff
@@ -92,7 +92,14 @@ packages:
       name: matcher
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.12.10"
+    version: "0.12.11"
+  material_color_utilities:
```

Comment:
Is this dependency necessary?

Reply:
It's a dependency by the Flutter SDK which have been added some months ago: https://github.com/flutter/flutter/pull/94377. I have not added any dependency. I just ran `flutter pub get` with the newest stable version in the example directory.

Reply:
Got it. Thanks.